cylc/uiserver/schema.py

```diff
@@ -747,33 +853,55 @@ def run_jobs_query(
 class UISTask(Task):
 
     platform = graphene.String()
+
     min_total_time = graphene.Int()
     mean_total_time = graphene.Int()
     max_total_time = graphene.Int()
     std_dev_total_time = graphene.Int()
-    queue_quartiles = graphene.List(
+    total_quartiles = graphene.List(
         graphene.Int,
         description=sstrip('''
-            List containing the first, second,
-            third and forth quartile queue times.'''))
+                List containing the first, second,
+                third and forth quartile total times.'''))
     min_queue_time = graphene.Int()
     mean_queue_time = graphene.Int()
     max_queue_time = graphene.Int()
     std_dev_queue_time = graphene.Int()
-    run_quartiles = graphene.List(
+    queue_quartiles = graphene.List(
         graphene.Int,
         description=sstrip('''
             List containing the first, second,
-            third and forth quartile run times.'''))
+            third and forth quartile queue times.'''))
     min_run_time = graphene.Int()
     mean_run_time = graphene.Int()
     max_run_time = graphene.Int()
     std_dev_run_time = graphene.Int()
-    total_quartiles = graphene.List(
+    run_quartiles = graphene.List(
         graphene.Int,
         description=sstrip('''
-            List containing the first, second,
-            third and forth quartile total times.'''))
+                List containing the first, second,
+                third and forth quartile run times.'''))
+    max_rss = graphene.String()
+    min_max_rss = graphene.Int()
+    mean_max_rss = graphene.Int()
+    max_max_rss = graphene.Int()
+    std_dev_max_rss = graphene.Int()
+    max_rss_quartiles = graphene.List(
+        graphene.Int,
+        description=sstrip('''
+                List containing the first, second,
+                third and forth quartile for Max RSS.'''))
+    min_cpu_time = graphene.Int()
+    mean_cpu_time = graphene.Int()
+    max_cpu_time = graphene.Int()
+    total_cpu_time = graphene.Int()
+    std_dev_cpu_time = graphene.Int()
```

Comment:
Actually shouldn't all these integers actually be floats?

Reply:
CPU time should be integers, but standard deviation should be a float, yes